web/templates/settings/upload_space_stats.hbs

```diff
@@ -1,9 +1,11 @@
 <span>
-    {{t "Organization using {percent_used}% of {upload_quota}." }}
     {{#if show_upgrade_message}}
-        {{#tr}}
-            <z-link>Upgrade</z-link> for more space.
-            {{#*inline "z-link"}}<a href="/upgrade/" target="_blank" rel="noopener noreferrer">{{> @partial-block }}</a>{{/inline}}
-        {{/tr}}
+    <a href="/upgrade/" class="space-tip" target="_blank" rel="noopener noreferrer">
+        {{t "Your organization is using {percent_used}% of your {upload_quota} file storage quota. Upgrade for more space." }}
```

Comment:
The translators would need to translate "Your organization is using .... " string twice. Maybe we can just pass translated string as an argument to template and then do `{{upload_quota_string}} {{t "Upgrade for more space"}}` in `#if show_upgrade_message` block and just `{{upload_quota_string}}` in else block.